runelite-client/src/main/java/net/runelite/client/plugins/fishing/FishingOverlay.java

```diff
@@ -54,6 +56,29 @@
 
 	private final PanelComponent panelComponent = new PanelComponent();
 
+	private final ImmutableSet<Integer> fishingAnimations = new ImmutableSet.Builder<Integer>()
```

Comment:
```suggestion
	private final ImmutableSet<Integer> fishingAnimations = new ImmutableSet.Builder<>()
```

Reply:
Type inference doesn't work here

Reply:
use https://guava.dev/releases/19.0/api/docs/com/google/common/collect/ImmutableSet.html#of(E,%20E,%20E,%20E,%20E,%20E,%20E...)

Reply:
You got it